packages/helpers/classes/mail.js

```diff
@@ -21,6 +21,7 @@ class Mail {
   constructor(data) {
 
     //Initialize array and object properties
+    this._isDynamic = false;
```

Comment:
I'm thinking just us `this.isDynamic` for the flag, e.g. without the underscore.